nautobot/ui/.gitignore

```diff
@@ -4,3 +4,6 @@ build
 # These files get rendered by `nautobot-server build_ui`
 app_imports.js
 jsconfig.paths.json
+
+# The output of `npm run analyze`
+analyze_report.html
```

Comment:
Are there docs on how to generate this report?

Reply:
There is an NPM command but hasn't been exposed as an invoke command yet.